src/bin.ts

```diff
@@ -101,7 +101,7 @@ if (mode === 'read') {
 
   getCurrentFuseWire(argv.app)
     .then((config) => {
-      const { version, resetAdHocDarwinSignature, ...rest } = config;
+      const { version, resetAdHocDarwinSignature, ignoreNotSupportedFuses, ...rest } = config;
```

Comment:
this appears unused.....what was the intent here?

Reply:
Well, here it is not used apparently. Removed also resetAdHocDarwingSignature as it is not used there as well.